force-app/main/default/classes/RelationshipModelGeneratorController.cls

```diff
@@ -243,6 +258,28 @@ public with sharing class RelationshipModelGeneratorController {
         set;
     }
 
+    private transient Set<SObjectType> extensibleSobTypes {
+        get {
+            if (extensibleSobTypes == null) {
+                extensibleSobTypes = new Set<SObjectType>();
+
+                Map<String, Schema.SObjectType> gd = Schema.getGlobalDescribe();
```

Comment:
It's probably better to use the `sobTypes` property, as `extensibleSobTypes` must be a subset of it